src/scene_battle_rpg2k.cpp

```diff
@@ -162,14 +162,16 @@ void Scene_Battle_Rpg2k::SetState(Scene_Battle::State new_state) {
 		break;
 	}
 
-	options_window->SetVisible(false);
-	status_window->SetVisible(false);
-	command_window->SetVisible(false);
-	item_window->SetVisible(false);
-	skill_window->SetVisible(false);
-	help_window->SetVisible(false);
-	target_window->SetVisible(false);
-	battle_message_window->SetVisible(false);
+	if (state != State_SelectEnemyTarget && state != State_SelectAllyTarget) {
+		options_window->SetVisible(false);
+		status_window->SetVisible(false);
+		command_window->SetVisible(false);
+		target_window->SetVisible(false);
+		battle_message_window->SetVisible(false);
+		item_window->SetVisible(false);
+		skill_window->SetVisible(false);
+		help_window->SetVisible(false);
+	}
```

Comment:
> commit "Fixes for select ally/target "

could you apply the whole commit to 2k3, code looks duplicated.

Reply:
I started looking at that and its not so simple. The behavior of battle windows for rm2k3 is different for the 3 battle modes. What is visible and what is not is different and the logic is tangled in many places outside of this function.

I'm going to push back on this. I don't want this PR getting bogged down on rm2k3 stuff. That should be a separate PR.